tests/capabilities/builders.py

```
@@ -0,0 +1,306 @@
+# Topology builders
+# =============================================================================
+
+
+def _build_linear(prefer_async: bool = False) -> Graph:
+    """Build A -> B -> C linear graph."""
+    if prefer_async:
+        return Graph([async_double_a, async_double_b], name="linear")
+    return Graph([double_a, double_b], name="linear")
+
+
+def _build_branching(prefer_async: bool = False) -> Graph:
```

Ruff (0.14.13)

Unused function argument: `prefer_async`

(ARG001)